manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1/CambridgeUniversityPress.CambridgeReader.yaml

```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: CambridgeUniversityPress.CambridgeReader
+PackageVersion: 1.0210.1

```

Comment:
```suggestion
PackageVersion: 1.0
```

Reply:
This needs to match Control Panel so `winget upgrade CambridgeUniversityPress.CambridgeReader` doesn't end up in a upgrade loop for version mismatch.

Reply:
You will also need to update the version folder as well.

Change `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0210.1` to `manifests/c/CambridgeUniversityPress/CambridgeReader/1.0`